api/Location.json

```diff
@@ -649,7 +649,7 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/history.html#dom-location-href-dev",
           "support": {
             "chrome": {
-              "version_added": "52"
+              "version_added": "29"
```

Comment:
This is not Firefox. Was it the Firefox data that was supposed to be updated to 29?

Reply:
No, Firefox is fine, I just selected the wrong browser option in my PR-creating script.  :P

Reply:
I've failed to confirm that Chrome 29 is correct using https://mdn-bcd-collector.appspot.com/tests/api/Location/toString. The test also passes on Chrome 28.